src/node_api_types.h

```diff
@@ -68,6 +75,9 @@ typedef enum {
   napi_name_expected,
   napi_function_expected,
   napi_number_expected,
+#ifdef NAPI_EXPERIMENTAL
```

Comment:
I don't think it's really necessary to gate them with NAPI_EXPERIMENTAL, just the new functions.

Do you agree @mhdawson @digitalinfinity @gabrielschulhof

Reply:
Perhaps we **must not** gate here because if two APIs (such as `thread_safe` and `bigint`) add new status codes while in experimental, they will have to come out of experimental in the order their added status codes appear, otherwise their emergence from experimental will itself constitute an ABI break, since their status codes would have to be moved before the bracket.

Reply:
/me go PR the removal of the gate now 🙂